tilt-resources/examples/deployment.yaml

```diff
@@ -0,0 +1,125 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    component: velero
+  name: velero
+  namespace: velero
+spec:
+  selector:
+    matchLabels:
+      deploy: velero
+  strategy: {}
+  template:
+    metadata:
+      annotations:
+        prometheus.io/path: /metrics
+        prometheus.io/port: "8085"
+        prometheus.io/scrape: "true"
+      labels:
+        component: velero
+        deploy: velero
+    spec:
+      containers:
+        - args:
+            - server 
+            - --log-level 
+            - debug       
+          command:
+            - /velero
+          env:
+            - name: VELERO_SCRATCH_DIR
+              value: /scratch
+            - name: VELERO_NAMESPACE
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.namespace
+            - name: LD_LIBRARY_PATH
+              value: /plugins
+            - name: AWS_SHARED_CREDENTIALS_FILE
+              value: /credentials/cloud
+            - name: AZURE_SHARED_CREDENTIALS_FILE
+              value: /credentials/cloud
+            - name: GOOGLE_APPLICATION_CREDENTIALS
+              value: /credentials/cloud
+          name: velero
+          image: velero/velero
+          imagePullPolicy: Always
+          ports:
+            - containerPort: 8085
+              name: metrics
+          resources:
+            limits:
+              cpu: "1"
+              memory: 256Mi
+            requests:
+              cpu: 500m
+              memory: 128Mi
+          volumeMounts:
+            - mountPath: /scratch
+              name: scratch
+            - mountPath: /plugins
+              name: plugins
+            - mountPath: /credentials
+              name: cloud-credentials
+      initContainers:
+        - image: velero/velero-plugin-for-aws
+          imagePullPolicy: Always
+          name: velero-plugin-for-aws
+          volumeMounts:
+          - mountPath: /target
+            name: plugins
+        - image: velero/velero-plugin-for-gcp
+          imagePullPolicy: Always
+          name: velero-plugin-for-gcp
+          volumeMounts:
+          - mountPath: /target
+            name: plugins
+        - image: velero/velero-plugin-for-microsoft-azure
+          imagePullPolicy: Always
+          name: velero-plugin-for-microsoft-azure
```

Comment:
csi plugin needs to be added here as well.

Reply:
I haven't watched the demo yet. But should all the plugins be added here?

Reply:
This is a sample file that people might choose to use as is, or delete things they don't want to run. I'm opting to offer more for the sake of better discovery, especially for new people onboarding.